CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the gem ibm-cloud-sdk-ruby will be documented here.
 
+## v0.1.7 - 2020-09-28
```

Comment:
Can you change this to `Unreleased - 2020-09-28` (we will change it when we cut the release, as the version may not be 0.1.7 if we need a minor or major instead)